src/server.ts

```diff
@@ -1,17 +1,19 @@
 import fs from 'fs';
 import path from 'path';
 import Hapi from '@hapi/hapi';
-import Joi from '@hapi/joi';
-import swaggered from 'hapi-swaggered';
-import swaggeredUI from 'hapi-swaggered-ui';
+import Joi from 'joi';
+import hapiSwagger from 'hapi-swagger';
+
 import vision from '@hapi/vision';
 import inert from '@hapi/inert';
 import requireHttps from 'hapi-require-https';
 import pino from 'hapi-pino';
 import json from 'fast-json-stringify';
 
 import checkApplicationHealth from './monitoring/health/routes';
-import { ServerArgs } from './';
+import { DbClient, Dict, ServerArgs } from './';
+
+type ServerType = Hapi.Server & { db?: void | typeof import('mongoose'); schema?: Dict };
```

Comment:
Same question about other dB types applies here also.

Reply:
Fixed db now points to DbClient as expected